### PR TITLE
Adding Bass and Treble settings

### PR DESCRIPTION
**Pull request by 3KUdelta**
I added the pass-through for the setTone() function.

### CODE REVIEW

**Comment by CelliesProjects** [comment]
Thanks for your effort! Looks good. It might take a couple of days before I can find the time to test and merge.

File: src/ESP32_VS1053_Stream.cpp [hunk 1]
```diff
@@ -417,6 +417,10 @@ void ESP32_VS1053_Stream::setVolume(const uint8_t vol) {
     if (_vs1053 && !_startMute) _vs1053->setVolume(_volume);
 }
 
+void ESP32_VS1053_Stream::setTone(uint8_t *rtone) {
+     _vs1053->setTone(rtone);
```
Comment (CelliesProjects) [inline src/ESP32_VS1053_Stream.cpp:421]: Please check if _vs1053 is not empty before

Comment (3KUdelta) [reply src/ESP32_VS1053_Stream.cpp:421]: Agree, I am using this currently without any problem. Do you think that there have to be some more checks?

Comment (CelliesProjects) [reply src/ESP32_VS1053_Stream.cpp:421]: I think just checking for a non NULL pointer to the hardware is enough.

```c++
if (_vs1053) _vs1053->setTone(rtone);
```

Comment (3KUdelta) [reply src/ESP32_VS1053_Stream.cpp:421]: Works for me - feel free to add.

**Comment by CelliesProjects** [comment]
It tests ok. One thing I would like to see before merging is the argument to the function as a  `const`.

**Comment by CelliesProjects** [comment]
Looking good. I will probably merge this PR coming weekend.
Thanks for your time and effort.

**Pull request merged**